src/AmbientSounds.Uwp/App.xaml.cs

```diff
@@ -37,6 +38,27 @@ public App()
                 // Ref: https://docs.microsoft.com/en-us/windows/uwp/xbox-apps/how-to-disable-mouse-mode
                 this.RequiresPointerMode = ApplicationRequiresPointerMode.WhenRequested;
             }
+
+            object themeObject = ApplicationData.Current.LocalSettings.Values["themeSetting"];
```

Comment:
The [localsettings](https://github.com/jenius-apps/ambie/blob/main/src/AmbientSounds.Uwp/Services/LocalSettings.cs) class can be used here instead of calling `applicationData.Current...` directly. 

Once I set up the settings viewmodel, I'll can show you how to use it